sms/twilio/checkBalance.js

```diff
@@ -0,0 +1,10 @@
+const call = require('./index');
```

Comment:
can we rename this to `sendSms` across the board? Just because it's a hackathon, doesn't mean we can't have good naming :)